docs/yaml/versions.yml

```diff
@@ -1,2 +1,3 @@
-version: 1.13.0
+version: 1.13.1
 quoteVersion: 0.4.1
+branch: relelase/v1.13
```

Comment:
what's this?

Reply:
I'm not sure, I was pretty blindly following directions, I guess I thought I was supposed to type that in at some point and clearly messed it up? Do you know what is supposed to be there?

Reply:
for now i don't think its hurting anyone, probably okay to merge then fixup in a subsequent PR just to get this repatriated.